tooling/lib/releases_diff.rb

```diff
@@ -33,12 +33,7 @@ def initialize(manifest_path = nil, output = nil)
         # Path to the current partial manifest
         @current_src_path=File.join(ReleasesDiff.git_root, current_manifest_path)
 
-        # Path to the anchors for the current stack
-        if "#{ENV['USE_SLE_BASE']}" == "false"
-            stack = ENV['FISSILE_LIGHT_OPEN42']
-        else
-            stack = ENV['FISSILE_LIGHT_SLE12']
-        end
+        stack = ENV["FISSILE_LIGHT_OPINIONS"]
```

Comment:
I know you simply removed the condition, but the label of this variable is very misleading IMO. How about renaming it and making it a const?
```ruby
FISSILE_LIGHT_OPINIONS = ENV["FISSILE_LIGHT_OPINIONS"]
```